.github/workflows/pip.yml

```diff
@@ -41,8 +41,8 @@ jobs:
     # Install dependencies 
     - name: Install
       run: |
-        python -m pip install numpy scipy matplotlib ${{matrix.pytest}}
-        python -m pip install -e .
+        python -m pip install ${{matrix.pytest}}
+        python -m pip install -e '.[test]'
```

Comment:
What is this syntax?

Reply:
And the groups are defined in pyproject.toml. Great, thanks for the explanation!